src/main/scala/main/Bot.scala

```diff
@@ -76,7 +76,9 @@ object Bot {
 
   def buildReviewStateStr(r: Review): String = {
     val slots            = 2
-    val qa: List[String] = (if (r.labels.map(_.name).contains("QA passed")) "🎨" else "⚪️") :: Nil
+    val qa: List[String] = (if (r.labels.map(_.name).contains("QA passed")) "🎨" 
```

Comment:
I used google and searched "scala if else if" and I think this works...but I dunno for sure. Please disregard if I'm a dumb dumb.